src/panels/lovelace/cards/hui-vertical-stack-card.ts

```diff
@@ -0,0 +1,87 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import computeCardSize from "../common/compute-card-size.js";
+import createCardElement from "../common/create-card-element.js";
+
+import { LovelaceCard, LovelaceConfig } from "../types.ts";
+
+interface Config extends LovelaceConfig {
+  cards: LovelaceConfig[];
+}
+
+class HuiVerticalStackCard extends LitElement
+  implements LovelaceCard {
+  protected config: Config;
+  private mHass: HomeAssistant;
+
+  static get properties() {
+    return {
+      config: {},
+    };
+  }
+
+  public getCardSize() {
+    let totalSize = 0;
+    for (const element of this.shadowRoot.querySelectorAll('#root > *')) {
+      totalSize += computeCardSize(element);
+    };
+
+    return totalSize;
+  }
+
+  public setConfig(config: Config) {
+    if (!config || !config.cards || !Array.isArray(config.cards)) {
+      throw new Error("Card config incorrect");
+    }
+
+    this.config = config;
+  }
+
+  protected render() {
+    if (!this.config) {
+      return html``;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <div id="root">
+        ${this.config.cards.map(card => createCardElement(card, this.mHass))}
```

Comment:
there is no second parameter?

Reply:
You'll need to do this above the return and then loop through it and set `el.hass = this._hass`

Reply:
The linter doesn't like that I'm trying to set `hass` on something of type `Element`. Is there a defined type we have that I should be casting it `as`? I was thinking `LovelaceCard` but doesn't look like that has `hass` in it.

Reply:
Also started seeing lint errors like this:
`Property 'config' has no initializer and is not definitely assigned in the constructor.`
I ran into something similar on another project with TS and ended up using classes with default initialization constructors.

Reply:
we want to convert createCardElement to typescript or cast it to a card:

```ts
const card = createCardElement(card) as LovelaceCard
```

Reply:
Make it `config?`. We don't initialize it in the constructor so should mark it as that it might not be available.